### DISCUSSION TITLE
Stripe Tax Support

### DISCUSSION THREAD

**Discussion by JosipDomazetDev** [Glench/ExtPay#96] [Ideas]
As extensions expand we will have to pay salex tax eventually. Stripe [offers](https://stripe.com/tax)
support to automtically take care of this. Adding stripe tax support to ExtPay will be a nice addition to future-proof it.

**Comment by bergwerf** [comment]
I've also been looking into this. Some countries have no sales tax exemption threshold at all, and I wonder if this means you would have to file taxes in every single country you have customers.

**Comment by bluecaret** [comment]
Anyone else already have experience dealing with taxes? This is something I'm very nervous about as I have no idea how it is supposed to work. I enabled the Stripe tax support but didn't realize that ExtPay also needs to support it for it to work, that would explain why no taxes have been collected yet. Some documentation about taxes for ExtPay would be nice too.
